Adding news post about my GRASS 8 video tutorials (#296)
OSGeo/grass-website: content/news/2022_03_22_New_Video_Tutorials_for_GRASS_8.md

---
title: "New Video Tutorials for GRASS GIS 8"
date: 2022-03-25
layout: "news"

> author: Isaac Ullah

Reviewer: ```suggestion
author: Isaac Ullah and the GRASS Dev Team

> title: "New Video Tutorials for GRASS 8"
> date: 2022-03-22
> layout: "news"
> author: Isaac Ullah
> ---
> 
> ### New video tutorials now available for GRASS version 8

Reviewer 2: ```suggestion
title: "New Video Tutorials for GRASS GIS 8"
date: 2022-03-25
layout: "news"
author: Isaac Ullah
---

### New video tutorials now available for GRASS GIS version 8
```

> Announcing a new series of publicly available YouTube video tutorials for GRASS 8. These tutorials, originally created for a GIS course at San Diego State University titled "ANTH 562 Computational Archaeology," are aimed to help novice to advanced GIS practioners learn how to use GRASS GIS. The subject matter is nominally archaeological GIS, but all the basics of GRASS are covered, as well as some advanced topics, such as Least Cost Analysis, Visibility Analysis, Hydrological Modeling, Terrain Analysis, and more. The sample data used in the tutorials is available in [this GitHub repository](https://github.com/isaacullah/GIS-Projects). 

Reviewer: ```suggestion
Announcing a new series of publicly available YouTube video tutorials for GRASS 8. These tutorials, originally created for a GIS course at San Diego State University titled "ANTH 562 Computational Archaeology", are aimed to help novice to advanced GIS practioners learn how to use GRASS GIS. The subject matter is nominally archaeological GIS, but all the basics of GRASS are covered, as well as some advanced topics, such as Least Cost Analysis, Visibility Analysis, Hydrological Modeling, Terrain Analysis, and more. The sample data used in the tutorials is available in [this GitHub repository](https://github.com/isaacullah/GIS-Projects). 
```

Author: Hi, yes I built it locally with Hugo, and it looked good. I can build and test again after all latest commits are merged.

The tutorials will be added to [this YouTube playlist](https://youtube.com/playlist?list=PLSCH2IXZ2pHrUXXuOK7EBKWmMD7i6CTA-), which is also embedded here for convenience:

> <iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?list=PLSCH2IXZ2pHrUXXuOK7EBKWmMD7i6CTA-" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

Reviewer: ```suggestion
<iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?list=PLSCH2IXZ2pHrUXXuOK7EBKWmMD7i6CTA-" title="GRASS GIS 8 Tutorial videos by Isaac Ullah" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Author: Sounds good!

These tutorials and sample data are released under a [CCbyNC 4.0](https://creativecommons.org/licenses/by-nc/4.0/) licence. You are free to use, remix, modify, or build on these data and content for any non-commercial purpose, but you must credit the original author.

#### About the author

The tutorials are created by Isaac I. Ullah, PhD. He is an associate professor of Anthropology at San Diego State University, specializing in computational approaches in archaeology. More information about Dr. Ullah's work is available on [his website](https://isaacullah.github.io/).

> *Isaac Ullah, Mar 2022*

Reviewer: ```suggestion

```
As the text is in the third person, I suggest to remove the signature.

Author: I will wait to commit until we can resolve the best approach. It does not seem correct to remove the post authorship from the YAML, since this is good metadata for future issues.

Author: I may be mistaken. I also run my own blog and website with a static site generator on github pages, and I know for sure that my site DOES add the author from the YAML. But, it uses Jekyll and probably has some other kinds of built-in formatting that this site may not have. It's possible that I add the author name since I copied the template from an earlier post, but I don't remember if I did that or not (too many things going on!). Either way, I am fine with changing the authorship to GRASS Dev Team. :)

Author: Oh! Yes, ok, that sounds fine to me.

Reviewer: This change remains pending; I'll apply and merge since you agreed
